<a href="https://colab.research.google.com/github/felipesayegg/Estatistica_com_Python/blob/main/Curso_de_Estat%C3%ADstica_Parte_2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

***
# <font color=green size=10>CURSO DE ESTATÍSTICA - PARTE 2</font>
***

# <font color=green>1 CONHECENDO OS DADOS</font>
***

## <font color=green>1.1 Dataset do projeto</font>
***

### Pesquisa Nacional por Amostra de Domicílios - 2015

A **Pesquisa Nacional por Amostra de Domicílios - PNAD** investiga anualmente, de forma permanente, características gerais da população, de educação, trabalho, rendimento e habitação e outras, com periodicidade variável, de acordo com as necessidades de informação para o país, como as características sobre migração, fecundidade, nupcialidade, saúde, segurança alimentar, entre outros temas. O levantamento dessas estatísticas constitui, ao longo dos 49 anos de realização da pesquisa, um importante instrumento para formulação, validação e avaliação de políticas orientadas para o desenvolvimento socioeconômico e a melhoria das condições de vida no Brasil.

### Fonte dos Dados

https://ww2.ibge.gov.br/home/estatistica/populacao/trabalhoerendimento/pnad2015/microdados.shtm

### Variáveis utilizadas

> ### Renda
> ***

Rendimento mensal do trabalho principal para pessoas de 10 anos ou mais de idade.

> ### Idade
> ***

Idade do morador na data de referência em anos.

> ### Altura (elaboração própria)
> ***

Altura do morador em metros.

> ### UF
> ***

|Código|Descrição|
|---|---|
|11|Rondônia|
|12|Acre|
|13|Amazonas|
|14|Roraima|
|15|Pará|
|16|Amapá|
|17|Tocantins|
|21|Maranhão|
|22|Piauí|
|23|Ceará|
|24|Rio Grande do Norte|
|25|Paraíba|
|26|Pernambuco|
|27|Alagoas|
|28|Sergipe|
|29|Bahia|
|31|Minas Gerais|
|32|Espírito Santo|
|33|Rio de Janeiro|
|35|São Paulo|
|41|Paraná|
|42|Santa Catarina|
|43|Rio Grande do Sul|
|50|Mato Grosso do Sul|
|51|Mato Grosso|
|52|Goiás|
|53|Distrito Federal|

> ### Sexo
> ***

|Código|Descrição|
|---|---|
|0|Masculino|
|1|Feminino|

> ### Anos de Estudo
> ***

|Código|Descrição|
|---|---|
|1|Sem instrução e menos de 1 ano|
|2|1 ano|
|3|2 anos|
|4|3 anos|
|5|4 anos|
|6|5 anos|
|7|6 anos|
|8|7 anos|
|9|8 anos|
|10|9 anos|
|11|10 anos|
|12|11 anos|
|13|12 anos|
|14|13 anos|
|15|14 anos|
|16|15 anos ou mais|
|17|Não determinados|
||Não aplicável|

> ### Cor
> ***

|Código|Descrição|
|---|---|
|0|Indígena|
|2|Branca|
|4|Preta|
|6|Amarela|
|8|Parda|
|9|Sem declaração|

#### <font color='red'>Observação</font>
***
> Os seguintes tratamentos foram realizados nos dados originais:
> 1. Foram eliminados os registros onde a **Renda** era inválida (999 999 999 999);
> 2. Foram eliminados os registros onde a **Renda** era missing;
> 3. Foram considerados somente os registros das **Pessoas de Referência** de cada domicílio (responsável pelo domicílio).

### Importando pandas e lendo o dataset do projeto

https://pandas.pydata.org/

In [13]:
import pandas as pd # importando a bibloteca pandas


In [14]:
dados = pd.read_csv('dados.csv') # dados vai levas o csv lido pela biblioteca pandas transformado em dataframe

In [15]:
dados.head()

UF  Sexo  Idade  Cor  Anos de Estudo  Renda    Altura
0  11     0     23    8              12    800  1.603808
1  11     1     23    2              12   1150  1.739790
2  11     1     35    8              15    880  1.760444
3  11     0     46    2               6   3500  1.783158
4  11     1     47    8               9    150  1.690631

---
---

# <font color=green>2 DISTRIBUIÇÕES DE PROBABILIDADE</font>
***

## <font color=red>Problema</font>
***

Em um concurso para preencher uma vaga de cientista de dados temos um total de **10 questões** de múltipla escolha com **3 alternativas possíveis** em cada questão. **Cada questão tem o mesmo valor.** Suponha que um candidato resolva se aventurar sem ter estudado absolutamente nada. Ele resolve fazer a prova de olhos vendados e chutar todas as resposta. Assumindo que a prova **vale 10 pontos e a nota de corte seja 5**, obtenha a probabilidade deste candidato **acertar 5 questões** e também a probabilidade deste candidato **passar para a próxima etapa do processo seletivo**.

## <font color=green>2.1 Distribuição Binomial</font>
***

Um evento **binomial** é caracterizado pela possibilidade de ocorrência de apenas duas categorias. Estas categorias somadas representam todo o espaço amostral, sendo também mutuamente excludentes, ou seja, a ocorrência de uma implica na não ocorrência da outra.

Em análises estatísticas o uso mais comum da distribuição binomial é na solução de problemas que envolvem situações de **sucesso** e **fracasso**.

# $$P(k)=\binom{n}{k} p^k q^{n-k}$$

Onde:

$p$ = probabilidade de sucesso - probalidade de obter cara em jogar uma moeda

$q = (1 - p)$ = probabilidade de fracasso - caso da moeda probalidade seria 50%

$n$ = número de eventos estudados

$k$ = número de eventos desejados que tenham sucesso

### Experimento Binomial

1. Realização de $n$ ensaios idênticos.

2. Os ensaios são independentes.

3. Somente dois resultados são possíveis, exemplo: Verdadeiro ou falso; Cara ou coroa; Sucesso ou fracasso.

4. A probabilidade de sucesso é representada por $p$ e a de fracasso por $1-p=q$. Estas probabilidades não se modificam de ensaio para ensaio.

### Média da distribuição binomial

O valor esperado ou a média da distribuição binomial é igual ao número de experimentos realizados multiplicado pela chance de ocorrência do evento.

# $$\mu = n \times p$$

### Desvio padrão da distribuição binomial

O desvio padrão é o produto entre o número de experimentos, a probabilidade de sucesso e a probabilidade de fracasso.

# $$\sigma = \sqrt{n \times p \times q}$$

### Importando bibliotecas
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.comb.html

In [16]:
from scipy.special import comb

### Combinações

Número de combinações de $n$ objetos, tomados $k$ a cada vez, é:

# $$C_{k}^{n} = \binom{n}{k} = \frac{n!}{k!(n - k)!}$$

Onde

## $$n! = n\times(n-1)\times(n-2)\times...\times(2)\times(1)$$
## $$k! = k\times(k-1)\times(k-2)\times...\times(2)\times(1)$$

Por definição

## $$0! = 1$$

## <font color='blue'>Exemplo: Mega Sena</font>

Em um volante de loteria da Mega Sena temos um total de **60 números** para escolher onde a aposta mínima é de **seis números**. Você que é curiosa(o) resolve calcular a probabilidade de se acertar na Mega Sena com apenas **um jogo**. Para isso precisamos saber quantas **combinações de seis números podem ser formadas com os 60 números disponíveis**.

### $$C_{6}^{60} = \binom{60}{6} = \frac{60!}{6!(60 - 6)!}$$

In [17]:
combinacoes = comb(60, 6)   # aqui esta fazendo um caclulo da probalidade de acerto da mega sena usando a biblioteca from scipy.special import comb
combinacoes

50063860.0

In [18]:
probabilidade = 1 / combinacoes
print("%0.15f" % probabilidade)

0.000000019974489


In [19]:
combinacoes_lotofacil = comb(25, 15)
combinacoes_lotofacil

3268760.0

In [20]:
probabilidade_lotofacil = 1 / combinacoes_lotofacil
print("%0.15f" % probabilidade_lotofacil)

0.000000305926406


In [21]:
alura_lota = comb(25, 20) # criando uma variavel e dentro dela colocando o valor de combinação sabendo que 25 é o nuemro de numeros e 20 e o numero que podemos selecionar dentro dos numeros
alura_lota

53130.0

In [22]:
prob_alura = 1 / alura_lota # calculando a probabilidade
print("%0.10f" % prob_alura)

0.0000188218


In [23]:
n = 4 # Numero de lançamentos da moeda
k = 2 #numero desejados de coroa

## <font color='blue'>Exemplo: Concurso para cientista de dados</font>

Em um concurso para preencher uma vaga de cientista de dados temos um total de **10 questões** de múltipla escolha com **3 alternativas possíveis** em cada questão. **Cada questão tem o mesmo valor.** Suponha que um candidato resolva se aventurar sem ter estudado absolutamente nada. Ele resolve fazer a prova de olhos vendados e chutar todas as resposta. Assumindo que a prova **vale 10 pontos e a nota de corte seja 5**, obtenha a probabilidade deste candidato **acertar 5 questões** e também a probabilidade deste candidato **passar para a próxima etapa do processo seletivo**.

### Qual o número de ensaios ($n$)?

In [24]:
#numero de questoes que tem na prova
n = 10
n

10

### Os ensaios são independentes?

Sim. A opção escolhida em uma questão não influencia em nada a opção escolhida em outra questão.

### Somente dois resultados são possíveis em cada ensaio?

Sim. O candidato tem duas possibilidades, ACERTA ou ERRAR uma questão.  Sabemos que tem 3 questoes mas so tem uma possibilidade errar ou acertar

### Qual a probabilidade de sucesso ($p$)?

In [25]:
#Sabendo que temos 3 alternativas e uma esta certa precisa ser 1 dividido por 3 que são numero de alternativa
numero_de_alternativas = 3
p = 1 / numero_de_alternativas
p

0.3333333333333333

### Qual a probabilidade de fracasso ($q$)?

In [26]:
q = 1 - p
# Seria 1 - p que é a probabilidade de acerto
q

0.6666666666666667

### Qual o total de eventos que se deseja obter sucesso ($k$)?

In [27]:
k = 5
k
#quantas questoes precisa acertar par aobter o sucesso

5

### Solução 1

In [28]:
probabilidade = (comb(n, k) * (p ** k) * (q ** (n - k)))
# basicamente pegamos a formula de Distribuição Binomial e aplicamos em cima dentro dessa formula tem outra que é a de combinação
print("%0.8f" % probabilidade)

0.13656455


### Importando bibliotecas
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.binom.html

In [29]:
from scipy.stats import binom

### Solução 2

In [30]:
probabilidade = binom.pmf(k, n, p) # foi usado a biblioteca binom que é uma otima aliada para quem quer calculcar probalidade binominal sem querer in afundo no calculos
print("%0.8f" % probabilidade)

0.13656455


### Obter a probabilidade do candidato passar

### $$P(acertar \geq 5) = P(5) + P(6) + P(7) + P(8) + P(9) + P10)$$

In [31]:
binom.pmf(5, n, p) + binom.pmf(6, n, p) + binom.pmf(7, n, p) + binom.pmf(8, n, p) + binom.pmf(9, n, p) +  binom.pmf(10, n, p)
#imagina que seria a probabilidade dele acertar 5 questoes depois 6 , 7,8,9,10 ai tem que somar todas par adar o valor certo dele passar na prova

0.21312808006909476

In [32]:
binom.pmf([5, 6, 7, 8, 9, 10], n, p).sum() #forma simplificada de fazer o codigo de cima

0.21312808006909476

In [33]:
binom.cdf(4, n, p)  # Aqui seria aprobalidade da pessoa acertar 1 questão + 2 , + 3 + 4

0.7868719199309048

In [34]:
binom.sf(4, n, p) # já aqui é a probalidade da pessao acertar 5,6,7,9,10 quesotes vai somar tudo

0.21312808006909525

Exatamente isso! Você entendeu perfeitamente!

binom.cdf(4, n, p): Aqui você está calculando a probabilidade de o candidato acertar até 4 questões. Isso significa que a função vai somar as probabilidades de ele acertar 0, 1, 2, 3 e 4 questões. Ou seja, é a soma de todas essas probabilidades.

binom.sf(4, n, p): Nesse caso, você está calculando a probabilidade de o candidato acertar mais do que 4 questões. Isso significa que a função vai considerar as probabilidades de ele acertar 5, 6, 7, até o máximo de questões que ele pode acertar (que é ( n )). Assim, ela soma todas essas probabilidades.

In [35]:
import math

# Variáveis
n = 4       # Número total de lançamentos
k = 2       # Número desejado de coroas
p = 0.5     # Probabilidade de coroa
q = 1 - p   # Probabilidade de não coroa

# Calcular o coeficiente binomial
combination = math.comb(n, k)
print(combination)

# Calcular a probabilidade binomial
probability = combination * (p ** k) * (q ** (n - k))



# Exibir o resultado
print(f"A probabilidade de obter exatamente {k} coroas em {n} lançamentos é {probability:.4f}")

6
A probabilidade de obter exatamente 2 coroas em 4 lançamentos é 0.3750


In [36]:
n = 4
p = 0.5
k = 2
q = 1 - p
probabilidade = (comb(n, k) * (p ** k) * (q ** (n - k)))
print("%0.8f" % probabilidade)

0.37500000


In [37]:
from scipy.stats import binom

# Variáveis
n = 4       # Número total de lançamentos
k = 2       # Número desejado de coroas
p = 0.5     # Probabilidade de obter coroa

# Calcular a probabilidade binomial usando scipy
probability = binom.pmf(k, n, p)

# Exibir o resultado
print(f"A probabilidade de obter exatamente {k} coroas em {n} lançamentos é {probability:.4f}")

A probabilidade de obter exatamente 2 coroas em 4 lançamentos é 0.3750


In [38]:
n = 10
k = 3
p = 1/6
probabilidade = binom.pmf([3,4, 5, 6, 7, 8, 9, 10], n, p).sum()
print("%0.8f" % probabilidade)

0.22477320


In [39]:
n = 10
k = 3
p = 1/6
q = 1 - p

provalidade_dado = binom.sf(2, n, p) #proalidade de cair masi que 2 vezes o lado 5 do dado ele soma de 3 ate 10
probabildiade_dado = binom.pmf([3, 4, 5, 6, 7, 8, 9, 10], n, p).sum()
probaçidade= binom.cdf(2, n, p)
print("%0.8f" % provalidade_dado)
provalidade_dado

0.22477320


0.22477320212874055

Definição: A Distribuição Binomial é utilizada para modelar experimentos que têm um número fixo de ensaios independentes, onde cada ensaio tem apenas dois resultados possíveis: sucesso ou fracasso.

Parâmetros:

n: número total de ensaios (ex: lançamentos da moeda).
k: número de sucessos desejados (ex: número de vezes que a face coroa aparece).
p: probabilidade de sucesso em cada ensaio (ex: para uma moeda justa, p = 0,5).
Funções:

pmf(k, n, p): calcula a probabilidade de obter exatamente k sucessos em n ensaios.
cdf(k, n, p): calcula a probabilidade de obter até k sucessos.
sf(k, n, p): calcula a probabilidade de obter mais do que k sucessos.
Exemplo: Se você lançar uma moeda 4 vezes e quer saber a probabilidade de obter 2 coroas, você usaria binom.pmf(2, 4, 0.5).

exercio seria basicamente a probailidade da pessoa acertar pelo menos 5 questão na prova e se resolve desse jeito acima

## <font color='blue'>Exemplo: Gincana</font>

Uma cidade do interior realiza todos os anos uma gincana para arrecadar fundos para o hospital da cidade. Na última gincana se sabe que a **proporção de participantes do sexo feminino foi de 60%**. **O total de equipes, com 12 integrantes, inscritas na gincana deste ano é de 30**. Com as informações acima responda: Quantas equipes deverão ser formadas por **8 mulheres**?

### Solução

In [40]:
k = 8
n = 12
p = 0.6
q = 1 - p
probabilidade = binom.pmf(k, n, p)
probabilidade

0.21284093952

In [41]:
equipes  = 30 * probabilidade
equipes

6.3852281856

Suponha que a probabilidade de um casal ter filhos com olhos azuis seja de 22%. Em 50 famílias, com 3 crianças cada uma, quantas podemos esperar que tenham dois filhos com olhos azuis?

In [42]:
p = 0.22
n = 3
k = 2
q = 1 - p
N = 50
probabilidade = binom.pmf(k, n, p) * N
probabilidade





5.662799999999996

---
---

## <font color=red>Problema</font>
***

Um restaurante recebe em média **20 pedidos por hora**. Qual a chance de que, em determinada hora escolhida ao acaso, o restaurante receba **15 pedidos**?

## <font color=green>2.2 Distribuição Poisson</font>
***

É empregada para descrever o número de ocorrências em um intervalo de tempo ou espaço específico. Os eventos são caracterizados pela possibilidade de contagem dos sucessos, mas a não possibilidade de contagem dos fracassos.

Como exemplos de processos onde podemos aplicar a distribuição de Poisson temos a determinação do número de clientes que entram em uma loja em determinada hora, o número de carros que chegam em um drive-thru de uma lanchonete na hora do almoço, a determinação do número de acidentes registrados em um trecho de estrada etc.

# $$P(k) = \frac{e^{-\mu}(\mu)^k}{k!}$$

Onde:

$e$ = constante cujo valor aproximado é 2,718281828459045

$\mu$ = representa o número médio de ocorrências em um determinado intervalo de tempo ou espaço

$k$ = número de sucessos no intervalo desejado

### Experimento Poisson

1. A probabilidade de uma ocorrência é a mesma em todo o intervalo observado.

2. O número de ocorrências em determinado intervalo é independente do número de ocorrências em outros intervalos.

3. A probabilidade de uma ocorrência é a mesma em intervalos de igual comprimento.

### Média da distribuição Poisson

# $$\mu$$

### Desvio padrão da distribuição Poisson

# $$\sigma = \sqrt{\mu}$$

### Importando bibliotecas

http://www.numpy.org/

In [43]:
import numpy as np

In [44]:
np.e

2.718281828459045

## <font color='blue'>Exemplo: Delivery</font>

Um restaurante recebe em média **20 pedidos por hora**. Qual a chance de que, em determinada hora escolhida ao acaso, o restaurante receba **15 pedidos**?

### Qual o número médio de ocorrências por hora ($\mu$)?

In [45]:
media = 20  # media de ocorrencia no inervalo de tempo que aqui no caso foi horas

### Qual o número de ocorrências que queremos obter no período ($k$)?

In [46]:
k = 15 # numero de eventos desejados que precisamos calcular

### Solução 1

In [47]:
probabilidade = ((np.e ** (-media)) * (media ** k)) / (np.math.factorial(k)) # jeito feito usando a formula mas existe uam função mais facil de realizar essa formula
print('%0.8f' % probabilidade)

0.05164885


<ipython-input-47-303616ca498c>:1: DeprecationWarning: `np.math` is a deprecated alias for the standard library `math` module (Deprecated Numpy 1.25). Replace usages of `np.math` with `math`
  probabilidade = ((np.e ** (-media)) * (media ** k)) / (np.math.factorial(k)) # jeito feito usando a formula mas existe uam função mais facil de realizar essa formula


### Importando bibliotecas

https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.poisson.html

### Solução 2

In [48]:
from scipy.stats import poisson # importanto a função da biblioteca scipy

probabilidade = poisson.pmf(k, media)  # formula para achar a probabilidade de um detrminado evento ocorrer em uma determinada hora prevista
# Leva 2 argumentos uma a media que ocorre esse evento e os eventos que queros saber

probabilidade

0.05164885353175814

O número médio de clientes que entram em uma padaria por hora é igual a 20. Obtenha a probabilidade de, na próxima hora, entrarem exatamente 25 clientes.

In [49]:
m = 20 # media de um evento acontecer em um determiando tempo no caso horas
k = 25 # numero de eventos que queremos saber a probabilidade

padaria = poisson.pmf(k, m)

padaria

0.04458764910826764

---
---

## <font color=red>Problema</font>
***

Em um estudo sobre as alturas dos moradores de uma cidade verificou-se que o conjunto de dados segue uma **distribuição aproximadamente normal**, com **média 1,70** e **desvio padrão de 0,1**. Com estas informações obtenha o seguinte conjunto de probabilidades:

> **A.** probabilidade de uma pessoa, selecionada ao acaso, ter menos de 1,80 metros.

> **B.** probabilidade de uma pessoa, selecionada ao acaso, ter entre 1,60 metros e 1,80 metros.    

> **C.** probabilidade de uma pessoa, selecionada ao acaso, ter mais de 1,90 metros.

## <font color=green>2.3 Distribuição Normal</font>
***

A distribuição normal é uma das mais utilizadas em estatística. É uma distribuição contínua, onde a distribuição de frequências de uma variável quantitativa apresenta a forma de sino e é simétrica em relação a sua média.

![Normal](https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img001.png)

### Características importantes

1. É simétrica em torno da média;

2. A área sob a curva corresponde à proporção 1 ou 100%;

3. As medidas de tendência central (média, mediana e moda) apresentam o mesmo valor;

4. Os extremos da curva tendem ao infinito em ambas as direções e, teoricamente, jamais tocam o eixo $x$;

5. O desvio padrão define o achatamento e largura da distribuição. Curvas mais largas e mais achatadas apresentam valores maiores de desvio padrão;

6. A distribuição é definida por sua média e desvio padrão;

7. A probabilidade sempre será igual à área sob a curva, delimitada pelos limites inferior e superior.

# $$f(x) = \frac{1}{\sqrt{2\pi\sigma}}e^{-\frac{1}{2}\left(\frac{x-\mu}{\sigma}\right)^2}$$

Onde:

$x$ = variável normal

$\sigma$ = desvio padrão

$\mu$ = média

A probabilidade é obtida a partir da área sob a curva, delimitada pelos limites inferior e superior especificados. Um exemplo pode ser visto na figura abaixo.

![alt text](https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img002.png)


Para obter a área acima basta calcular a integral da função para os intervalos determinados. Conforme equação abaixo:

# $$P(L_i<x<L_s) = \int_{L_i}^{L_s}\frac{1}{\sqrt{2\pi\sigma}}e^{-\frac{1}{2}\left(\frac{x-\mu}{\sigma}\right)^2}$$

Onde:

$x$ = variável normal

$\sigma$ = desvio padrão

$\mu$ = média

$L_i$ = limite inferior

$L_s$ = limite superior

### Tabelas padronizadas

As tabelas padronizadas foram criadas para facilitar a obtenção dos valores das áreas sob a curva normal e eliminar a necessidade de solucionar integrais definidas.

Para consultarmos os valores em uma tabela padronizada basta transformarmos nossa variável em uma variável padronizada $Z$.

Esta variável $Z$ representa o afastamento em desvios padrões de um valor da variável original em relação à média.

# $$Z = \frac{x-\mu}{\sigma}$$

Onde:

$x$ = variável normal com média $\mu$ e desvio padrão $\sigma$

$\sigma$ = desvio padrão

$\mu$ = média

### Construindo tabela normal padronizada
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html

In [50]:
import pandas as pd
import numpy as np
from scipy.stats import norm

tabela_normal_padronizada = pd.DataFrame(
    [],
    index=["{0:0.2f}".format(i / 100) for i in range(0, 400, 10)],
    columns = ["{0:0.2f}".format(i / 100) for i in range(0, 10)])

for index in tabela_normal_padronizada.index:
    for column in tabela_normal_padronizada.columns:
        Z = np.round(float(index) + float(column), 2)
        tabela_normal_padronizada.loc[index, column] = "{0:0.4f}".format(norm.cdf(Z))

tabela_normal_padronizada.rename_axis('Z', axis = 'columns', inplace = True)

tabela_normal_padronizada

Z       0.00    0.01    0.02    0.03    0.04    0.05    0.06    0.07    0.08  \
0.00  0.5000  0.5040  0.5080  0.5120  0.5160  0.5199  0.5239  0.5279  0.5319   
0.10  0.5398  0.5438  0.5478  0.5517  0.5557  0.5596  0.5636  0.5675  0.5714   
0.20  0.5793  0.5832  0.5871  0.5910  0.5948  0.5987  0.6026  0.6064  0.6103   
0.30  0.6179  0.6217  0.6255  0.6293  0.6331  0.6368  0.6406  0.6443  0.6480   
0.40  0.6554  0.6591  0.6628  0.6664  0.6700  0.6736  0.6772  0.6808  0.6844   
0.50  0.6915  0.6950  0.6985  0.7019  0.7054  0.7088  0.7123  0.7157  0.7190   
0.60  0.7257  0.7291  0.7324  0.7357  0.7389  0.7422  0.7454  0.7486  0.7517   
0.70  0.7580  0.7611  0.7642  0.7673  0.7704  0.7734  0.7764  0.7794  0.7823   
0.80  0.7881  0.7910  0.7939  0.7967  0.7995  0.8023  0.8051  0.8078  0.8106   
0.90  0.8159  0.8186  0.8212  0.8238  0.8264  0.8289  0.8315  0.8340  0.8365   
1.00  0.8413  0.8438  0.8461  0.8485  0.8508  0.8531  0.8554  0.8577  0.8599   
1.10  0.8643  0.8665  0.8686  0.8708  0.8729  0.8749  0.8770  0.8790  0.8810   
1.20  0.8849  0.8869  0.8888  0.8907  0.8925  0.8944  0.8962  0.8980  0.8997   
1.30  0.9032  0.9049  0.9066  0.9082  0.9099  0.9115  0.9131  0.9147  0.9162   
1.40  0.9192  0.9207  0.9222  0.9236  0.9251  0.9265  0.9279  0.9292  0.9306   
1.50  0.9332  0.9345  0.9357  0.9370  0.9382  0.9394  0.9406  0.9418  0.9429   
1.60  0.9452  0.9463  0.9474  0.9484  0.9495  0.9505  0.9515  0.9525  0.9535   
1.70  0.9554  0.9564  0.9573  0.9582  0.9591  0.9599  0.9608  0.9616  0.9625   
1.80  0.9641  0.9649  0.9656  0.9664  0.9671  0.9678  0.9686  0.9693  0.9699   
1.90  0.9713  0.9719  0.9726  0.9732  0.9738  0.9744  0.9750  0.9756  0.9761   
2.00  0.9772  0.9778  0.9783  0.9788  0.9793  0.9798  0.9803  0.9808  0.9812   
2.10  0.9821  0.9826  0.9830  0.9834  0.9838  0.9842  0.9846  0.9850  0.9854   
2.20  0.9861  0.9864  0.9868  0.9871  0.9875  0.9878  0.9881  0.9884  0.9887   
2.30  0.9893  0.9896  0.9898  0.9901  0.9904  0.9906  0.9909  0.9911  0.9913   
2.40  0.9918  0.9920  0.9922  0.9925  0.9927  0.9929  0.9931  0.9932  0.9934   
2.50  0.9938  0.9940  0.9941  0.9943  0.9945  0.9946  0.9948  0.9949  0.9951   
2.60  0.9953  0.9955  0.9956  0.9957  0.9959  0.9960  0.9961  0.9962  0.9963   
2.70  0.9965  0.9966  0.9967  0.9968  0.9969  0.9970  0.9971  0.9972  0.9973   
2.80  0.9974  0.9975  0.9976  0.9977  0.9977  0.9978  0.9979  0.9979  0.9980   
2.90  0.9981  0.9982  0.9982  0.9983  0.9984  0.9984  0.9985  0.9985  0.9986   
3.00  0.9987  0.9987  0.9987  0.9988  0.9988  0.9989  0.9989  0.9989  0.9990   
3.10  0.9990  0.9991  0.9991  0.9991  0.9992  0.9992  0.9992  0.9992  0.9993   
3.20  0.9993  0.9993  0.9994  0.9994  0.9994  0.9994  0.9994  0.9995  0.9995   
3.30  0.9995  0.9995  0.9995  0.9996  0.9996  0.9996  0.9996  0.9996  0.9996   
3.40  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997  0.9997   
3.50  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998  0.9998   
3.60  0.9998  0.9998  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999   
3.70  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999   
3.80  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999  0.9999   
3.90  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000  1.0000   

Z       0.09  
0.00  0.5359  
0.10  0.5753  
0.20  0.6141  
0.30  0.6517  
0.40  0.6879  
0.50  0.7224  
0.60  0.7549  
0.70  0.7852  
0.80  0.8133  
0.90  0.8389  
1.00  0.8621  
1.10  0.8830  
1.20  0.9015  
1.30  0.9177  
1.40  0.9319  
1.50  0.9441  
1.60  0.9545  
1.70  0.9633  
1.80  0.9706  
1.90  0.9767  
2.00  0.9817  
2.10  0.9857  
2.20  0.9890  
2.30  0.9916  
2.40  0.9936  
2.50  0.9952  
2.60  0.9964  
2.70  0.9974  
2.80  0.9981  
2.90  0.9986  
3.00  0.9990  
3.10  0.9993  
3.20  0.9995  
3.30  0.9997  
3.40  0.9998  
3.50  0.9998  
3.60  0.9999  
3.70  0.9999  
3.80  0.9999  
3.90  1.0000

<img src='https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img003.png' width='250px'>

A tabela acima fornece a área sob a curva entre $-\infty$ e $Z$ desvios padrão acima da média. Lembrando que por se tratar de valores padronizados temos $\mu = 0$.

## <font color='blue'>Exemplo: Qual sua altura?</font>

Em um estudo sobre as alturas dos moradores de uma cidade verificou-se que o conjunto de dados segue uma **distribuição aproximadamente normal**, com **média 1,70** e **desvio padrão de 0,1**. Com estas informações obtenha o seguinte conjunto de probabilidades:

> **A.** probabilidade de uma pessoa, selecionada ao acaso, ter menos de 1,80 metros.

> **B.** probabilidade de uma pessoa, selecionada ao acaso, ter entre 1,60 metros e 1,80 metros.    

> **C.** probabilidade de uma pessoa, selecionada ao acaso, ter mais de 1,90 metros.

### Problema A - Identificação da área sob a curva

<img style='float: left' src='https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img004.png' width='350px'>

### Obter a variável padronizada $Z$

In [51]:
media = 1.70
media

1.7

In [76]:
desvio_padrao = 0.1
desvio_padrao

0.1

In [53]:
Z = (1.8 - media) / desvio_padrao
Z

1.0000000000000009

### Solução 1 - Utilizando tabela

In [54]:
probabilidade = 0.8413 # dado o valor q la em cima na formula de normalizar para z eu preciso ir na tabela e achar um para achar a probailidade e responder a pergunta do meu problema

### Solução 2 - Utilizando Scipy

In [55]:
from scipy.stats import norm
norm.cdf(Z) # imagina que com essa função eu não preciso ir na tabela e achar a probabildiade baseado no z


0.8413447460685431

A aplicação de uma prova de estatística em um concurso apresentou um conjunto de notas normalmente distribuídas. Verificou-se que o conjunto de notas tinha média 70 e desvio padrão de 5 pontos.

Qual a probabilidade de um aluno, selecionado ao acaso, ter nota menor que 85?

In [56]:
media_alunos = 70  # media do banco de dados
desvio_alunos = 5 # desvio da nota

z_alunos = (85-media_alunos) / desvio_alunos # formula para achar o z e assim usar aa função sabendo que 85 pra baixo queremos saber a probabildiad eda pessoa tirar isso
z_alunos


norm.cdf(z_alunos)

0.9986501019683699

### Problema B - Identificação da área sob a curva

<img style='float: left' src='https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img005.png' width='350px'>

### Obter a variável padronizada $Z$

In [57]:
Z = (1.7 - media) / desvio_padrao
Z

0.0

In [58]:
Z = (1.8 - media) / desvio_padrao
Z

1.0000000000000009

### Solução 1 - Utilizando tabela

In [59]:
probabilidade = 0.8413 - 0.5
probabilidade

0.34130000000000005

In [60]:
probabilidade = (0.8413 - 0.5) * 2
probabilidade

0.6826000000000001

### Solução 2 - Utilizando Scipy

In [61]:
Z_superior = (1.8 - media) / desvio_padrao
Z_superior

1.0000000000000009

In [62]:
Z_inferior = (1.6 - media) / desvio_padrao
round(Z_inferior, 2)

-1.0

In [63]:
Z_superior = (1.8 - media) / desvio_padrao
round(Z_superior, 2)

1.0

In [64]:
probabilidade = norm.cdf(Z_superior)
probabilidade

0.8413447460685431

In [65]:
probabilidade = norm.cdf(Z_superior) - (1- norm.cdf(Z_superior))

In [66]:
probabilidade = 0.8413 - (1 - 0.8413)
probabilidade

0.6826000000000001

In [67]:
probabilidade = norm.cdf(Z_superior) - norm.cdf(Z_inferior)
probabilidade

0.6826894921370857

In [68]:
from scipy.stats import norm

media = 300
desvio_padrao = 50
Z_inferior = (250 - media) / desvio_padrao
Z_superior = (350 - media) / desvio_padrao

probabilidade = norm.cdf(Z_superior) - norm.cdf(Z_inferior)
print("{0:.2%}".format(probabilidade))

68.27%


In [69]:
from scipy.stats import norm

media = 300
desvio_padrao = 50
Z_inferior = (400 - media) / desvio_padrao
Z_superior = (500 - media) / desvio_padrao

probabilidade = norm.cdf(Z_superior) - norm.cdf(Z_inferior)
print("{0:.2%}".format(probabilidade))

2.27%


O faturamento diário de um motorista de aplicativo segue uma distribuição aproximadamente normal, com média R$ 300,00 e desvio padrão igual a R$ 50,00. Obtenha as probabilidades de que, em um dia aleatório, o motorista ganhe:

1) Entre R$ 250,00 e R$ 350,00

2) Entre R$ 400,00 e R$ 500,00

In [70]:
media_uber = 300
desvio_uber = 50

z_uber = (350 - media_uber) / desvio_uber
z_uber

1.0

In [71]:
probabildiade_350 = norm.cdf(z_uber)
probabildiade_350   # de fazer ate 350 pra baixo essa é a probabilidade

0.8413447460685429

In [72]:
probabilidade = norm.cdf(z_uber) - (1 - norm.cdf(z_uber))  # aqui eu quero saber a probailidade entre 250 a 350 1 - z_uber eu ja tenho a metade que faltou pra cima e o z_uber ja tirou uma metade tambem então fazenod esse calculo da a probabilidade
probabilidade

0.6826894921370859

### Problema C - Identificação da área sob a curva

<img style='float: left' src='https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img006.png' width='350px'>

### Obter a variável padronizada $Z$

In [82]:
desvio_padrao = 0.1
media = 1.70
Z = (1.9 - media) / desvio_padrao
Z


1.9999999999999996

In [83]:
probabilidade = 1 - 0.9767  # Lembrando que 0.97 eu tirei da tabela pelo alor dado do z
# Lembrando que esse valor de 0.97 probabilidade de uma pessoa ter ate 1.9 então o exercico quer saber mais por isso menos 1
probabilidade

0.023299999999999987

### Solução 1 - Utilizando tabela

### Solução 2 - Utilizando Scipy

In [85]:
probabilidade = 1 - norm.cdf(Z)  # foi feito usando cdf função onde ele busca na tabela dado o valor de z

probabilidade

0.02275013194817921

In [86]:
probabilidade = norm.cdf(-Z)  # cdf é espero o bastante para entender se eu estou colocndo menos ele não quer saber a area a esqueda dos dados e sim a direita como pode exercicio
probabilidade

0.022750131948179216

O Inmetro verificou que as lâmpadas incandescentes da fabricante XPTO apresentam uma vida útil normalmente distribuída, com média igual a 720 dias e desvio padrão igual a 30 dias. Calcule a probabilidade de uma lâmpada, escolhida ao acaso, durar:

1) Entre 650 e 750 dias

2) Mais que 800 dias

3) Menos que 700 dias

In [88]:
media_lampada = 720 # media lampada dada no exercicio
desvio_lampada = 30  # desvio dessa media tanto para cima e para baixo



In [92]:
z_lampada =(750 - media_lampada) / desvio_lampada  # formula para achar o valor do z para usar a função
z_lampada



1.0

In [93]:
z_lampada2 = (650 - media_lampada) / desvio_lampada # achando o valor do z para ate 650 dias
z_lampada2

-2.3333333333333335

In [99]:
norm.cdf(z_lampada) # aqui seria de 750 para baixo

0.8413447460685429

In [101]:
norm.cdf(z_lampada2) aqui seria os 650 para baixo

0.009815328628645334

In [103]:
norm.cdf(z_lampada) - norm.cdf(z_lampada2)  # eu entendo que para saber o intervalo so temos que tirar a probabildiae dos 650 dia para baixo do valor de 750 que vai se encaixa rno intervalo de 650 ate 750

0.8315294174398976

In [97]:
probabilidade_a = norm.cdf(z_lampada) - norm.cdf(z_lampada2)
probabilidade_a

0.8315294174398976

In [105]:
# Calcula o Z-score para o valor 800 em relação à média e desvio padrão das medições da lâmpada
z_lampada3 = (800 - media_lampada) / desvio_lampada

# Exibe o valor do Z-score calculado
z_lampada3

# Calcula a probabilidade de uma medição ser maior que o Z-score calculado
# norm.cdf(z_lampada3) retorna a probabilidade acumulada até z_lampada3
# Subtraímos esse valor de 1 para obter a probabilidade complementar
probabilidade_b = 1 - norm.cdf(z_lampada3)

# Exibe a probabilidade calculada
probabilidade_b

0.003830380567589775

In [107]:
# Calcula o Z-score para o valor 700 em relação à média e desvio padrão das medições da lâmpada
z_lampada4 = (700 - media_lampada) / desvio_lampada

# Exibe o valor do Z-score calculado para 700
z_lampada4

# Calcula a probabilidade de uma medição ser menor ou igual ao Z-score calculado
# norm.cdf(z_lampada4) retorna a probabilidade acumulada até z_lampada4
probabilidade_c = norm.cdf(z_lampada4)

# Exibe a probabilidade calculada
probabilidade_c

0.2524925375469229

Utilizando a tabela padronizada, ou o ferramental disponibilizado pelo Python, encontre a área sob a curva normal para os valores de Z abaixo:

1) Z < 1,96

2) Z > 2,15

3) Z < -0,78

4) Z > 0,59

In [112]:
z_one = norm.cdf(1.96)
z_one

0.9750021048517795

In [115]:
z_two = norm.cdf(-2.15)
z_two

0.0157776073910905

In [116]:
z_three = norm.cdf(-0.78)
z_three

0.21769543758573312

In [118]:
z_four = norm.cdf(-0.59)
z_four

0.27759532475346493

# <font color=green>3 AMOSTRAGEM</font>
***

## <font color=green>3.1 População e Amostra</font>
***

### População
Conjunto de todos os elementos de interesse em um estudo. Diversos elementos podem compor uma população, por exemplo: pessoas, idades, alturas, carros etc.

Com relação ao tamanho, as populações podem ser limitadas (populações finitas) ou ilimitadas (populações infinitas).

### Populações finitas

Permitem a contagem de seus elementos. Como exemplos temos o número de funcionário de uma empresa, a quantidade de alunos em uma escola etc.

### Populações infinitas

Não é possível contar seus elementos. Como exemplos temos a quantidade de porções que se pode extrair da água do mar para uma análise, temperatura medida em cada ponto de um território etc.

<font color=red>Quando os elementos de uma população puderem ser contados, porém apresentando uma quantidade muito grande, assume-se a população como infinita.</font>.

### Amostra
Subconjunto representativo da população.

Os atributos numéricos de uma população como sua média, variância e desvio padrão, são conhecidos como **parâmetros**. O principal foco da inferência estatística é justamente gerar estimativas e testar hipóteses sobre os parâmetros populacionais utilizando as informações de amostras.

## <font color=green>3.2 Quando utilizar uma amostra?</font>
***

### Populações infinitas

O estudo não chegaria nunca ao fim. Não é possível investigar todos os elementos da população.

### Testes destrutivos

Estudos onde os elementos avaliados são totalmente consumidos ou destruídos. Exemplo: testes de vida útil, testes de segurança contra colisões em automóveis.

### Resultados rápidos

Pesquisas que precisam de mais agilidade na divulgação. Exemplo: pesquisas de opinião, pesquisas que envolvam problemas de saúde pública.

### Custos elevados

Quando a população é finita mas muito numerosa, o custo de um censo pode tornar o processo inviável.

## <font color=green>3.3 Amostragem Aleatória Simples</font>
***

É uma das principais maneiras de se extrair uma amostra de uma população. A exigência fundamental deste tipo de abordagem é que cada elemeto da população tenha as mesmas chances de ser selecionado para fazer parte da amostra.

In [124]:
dados.shape[0] # quantas linhas

76840

In [125]:
dados.shape[1] # Quantas colunas meu dataframe

7

In [126]:
dados.Renda.mean()  # Vim em dados peguei a variavel renda e tirei a media

2000.3831988547631

In [127]:
amostra = dados.sample(n = 100, random_state = 101) # Criei uma amostra  com a função sample
amostra

UF  Sexo  Idade  Cor  Anos de Estudo  Renda    Altura
29042  29     0     39    8               5    480  1.719128
62672  43     0     55    2               6    250  1.639205
29973  29     1     36    2              12    788  1.654122
22428  26     0     46    8               8   1680  1.622450
55145  41     0     37    2               9   2500  1.625268
...    ..   ...    ...  ...             ...    ...       ...
40245  32     0     35    2              12   2800  1.668898
30997  29     1     34    4              14   1924  1.795315
15094  23     0     60    4               1    130  1.682966
48788  35     1     39    2               7    460  1.777723
34891  31     0     42    2               3   1200  1.764392

[100 rows x 7 columns]

In [128]:
amostra.shape[0]

100

In [129]:
amostra.Renda.mean() # minha amostra ocorreu uma variação da media de salario

2150.85

In [133]:
dados.Sexo.value_counts(normalize = True)  # 0 proporção de masculino e feminino

Sexo
0    0.692998
1    0.307002
Name: proportion, dtype: float64

In [134]:
amostra.Sexo.value_counts(normalize = True)  # amostra esta super parecido com os dados da populção

Sexo
0    0.69
1    0.31
Name: proportion, dtype: float64

## <font color=green>3.4 Amostragem Estratificada</font>
***

É uma melhoria do processo de amostragem aleatória simples. Neste método é proposta a divisão da população em subgrupos de elementos com características similares, ou seja, grupos mais homogêneos. Com estes subgrupos separados, aplica-se a técnica de amostragem aleatória simples dentro de cada subgrupo individualmente.

## <font color=green>3.5 Amostragem por Conglomerados</font>
***

Também visa melhorar o critério de amostragem aleatória simples. Na amostragem por conglomerados são também criados subgrupos, porém não serão homogêneas como na amostragem estratificada. Na amostragem por conglomerados os subgrupos serão heterogêneos, onde, em seguida, serão aplicadas a amostragem aleatória simples ou estratificada.

Um exemplo bastante comum de aplicação deste tipo de técnica é na divisão da população em grupos territoriais, onde os elementos investigados terão características bastante variadas.

# <font color=green>4 ESTIMAÇÃO</font>
***

## <font color='red'>Problema </font>

Suponha que os pesos dos sacos de arroz de uma indústria alimentícia se distribuem aproximadamente como uma normal de **desvio padrão populacional igual a 150 g**. Selecionada uma **amostra aleatório de 20 sacos** de um lote específico, obteve-se um **peso médio de 5.050 g**. Construa um **intervalo de confiança para a média populacional** assumindo um **nível de significância de 5%**.

---

É a forma de se fazer suposições generalizadas sobre os parâmetros de uma população tendo como base as informações de uma amostra.

- **Parâmetros** são os atributos numéricos de uma população, tal como a média, desvio padrão etc.

- **Estimativa** é o valor obtido para determinado parâmetro a partir dos dados de uma amostra da população.

## <font color=green>4.1 Teorema do limite central</font>
***

> O **Teorema do Limite Central** afirma que, com o aumento do tamanho da amostra, a distribuição das médias amostrais se aproxima de uma distribuição normal com média igual à média da população e desvio padrão igual ao desvio padrão da variável original dividido pela raiz quadrada do tamanho da amostra. Este fato é assegurado para $n$ maior ou igual a 30.

# $$\sigma_\bar{x} = \frac{\sigma}{\sqrt{n}}$$

O desvio padrão das médias amostrais é conhecido como **erro padrão da média**

### Entendendo o Teorema do Limite Central

> O Teorema do Limite Central afirma que, **com o aumento do tamanho da amostra, a distribuição das médias amostrais se aproxima de uma distribuição normal** com média igual à média da população e desvio padrão igual ao desvio padrão da variável original dividido pela raiz quadrada do tamanho da amostra. Este fato é assegurado para n maior ou igual a 30.

> O Teorema do Limite Central afirma que, com o aumento do tamanho da amostra, a distribuição das médias amostrais se aproxima de uma distribuição normal **com média igual à média da população** e desvio padrão igual ao desvio padrão da variável original dividido pela raiz quadrada do tamanho da amostra. Este fato é assegurado para n maior ou igual a 30.

> O Teorema do Limite Central afirma que, com o aumento do tamanho da amostra, a distribuição das médias amostrais se aproxima de uma distribuição normal com média igual à média da população e **desvio padrão igual ao desvio padrão da variável original dividido pela raiz quadrada do tamanho da amostra**. Este fato é assegurado para n maior ou igual a 30.

# $$\sigma_\bar{x} = \frac{\sigma}{\sqrt{n}}$$

## <font color=green>4.2 Níveis de confiança e significância</font>
***

O **nível de confiança** ($1 - \alpha$) representa a probabilidade de acerto da estimativa. De forma complementar o **nível de significância** ($\alpha$) expressa a probabilidade de erro da estimativa.

O **nível de confiança** representa o grau de confiabilidade do resultado da estimativa estar dentro de determinado intervalo. Quando fixamos em uma pesquisa um **nível de confiança** de 95%, por exemplo, estamos assumindo que existe uma probabilidade de 95% dos resultados da pesquisa representarem bem a realidade, ou seja, estarem corretos.

O **nível de confiança** de uma estimativa pode ser obtido a partir da área sob a curva normal como ilustrado na figura abaixo.

![alt text](https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img007.png)

## <font color=green>4.3 Erro inferencial</font>
***

O **erro inferencial** é definido pelo **desvio padrão das médias amostrais** $\sigma_\bar{x}$ e pelo **nível de confiança** determinado para o processo.

# $$e = z \frac{\sigma}{\sqrt{n}}$$

## <font color=green>4.4 Intervalos de confiança</font>
***

### Intevalo de confiança para a média da população

#### Com desvio padrão populacional conhecido

## $$\mu = \bar{x} \pm z\frac{\sigma}{\sqrt{n}}$$

#### Com desvio padrão populacional desconhecido

## $$\mu = \bar{x} \pm z\frac{s}{\sqrt{n}}$$

## <font color='blue'>Exemplo: </font>

Suponha que os pesos dos sacos de arroz de uma indústria alimentícia se distribuem aproximadamente como uma normal de **desvio padrão populacional igual a 150 g**. Selecionada uma **amostra aleatório de 20 sacos** de um lote específico, obteve-se um **peso médio de 5.050 g**. Construa um intervalo de confiança para a **média populacional** assumindo um **nível de significância de 5%**.

### Média amostral

### Nível de significância ($\alpha$)

### Nível de confiança ($1 - \alpha$)

### Obtendo $z$

### Obtendo $z$

![alt text](https://caelum-online-public.s3.amazonaws.com/1178-estatistica-parte2/01/img008.png)

### Valores de $z$ para os níveis de confiança mais utilizados

|Nível de<br>confiança|Valor da área sob<br>a curva normal| $z$ |
|:----------------:|:---------------------------------:|:---:|
|90%               |0,95                               |1,645|
|95%               |0,975                              |1,96 |
|99%               |0,995                              |2,575|

### Obtendo $\sigma_\bar{x}$

### Obtendo $e$

### Solução 1 - Calculando o intervalo de confiança para a média

### Solução 2 - Calculando o intervalo de confiança para a média

# <font color=green>5 CÁLCULO DO TAMANHO DA AMOSTRA</font>
***

## <font color='red'>Problema </font>

Estamos estudando o rendimento mensal dos chefes de domicílios com renda até R$\$$ 5.000,00 no Brasil. Nosso supervisor determinou que o **erro máximo em relação a média seja de R$\$$ 10,00**. Sabemos que o **desvio padrão populacional** deste grupo de trabalhadores é de **R$\$$ 1.082,79**. Para um **nível de confiança de 95%**, qual deve ser o tamanho da amostra de nosso estudo?

## <font color=green>5.1 Variáveis quantitativas e população infinita</font>
***

# $$e = z \frac{\sigma}{\sqrt{n}}$$

#### Com desvio padrão conhecido

## $$n = \left(z\frac{\sigma}{e}\right)^2$$

#### Com desvio padrão desconhecido

## $$n = \left(z\frac{s}{e}\right)^2$$

Onde:

$z$ = variável normal padronizada

$\sigma$ = desvio padrão populacional

$s$ = desvio padrão amostral

$e$ = erro inferencial

### <font color='red'>Observações</font>

1. O desvio padrão ($\sigma$ ou $s$) e o erro ($e$) devem estar na mesma unidade de medida.

2. Quando o erro ($e$) for representado em termos percentuais, deve ser interpretado como um percentual relacionado à média.

## <font color='blue'>Exemplo: Rendimento médio</font>

Estamos estudando o rendimento mensal dos chefes de domicílios no Brasil. Nosso supervisor determinou que o **erro máximo em relação a média seja de R$\$$ 100,00**. Sabemos que o **desvio padrão populacional** deste grupo de trabalhadores é de **R$\$$ 3.323,39**. Para um **nível de confiança de 95%**, qual deve ser o tamanho da amostra de nosso estudo?

### Obtendo $\sigma$

### Obtendo $e$

### Obtendo $n$

---
---

## <font color='red'>Problema</font>

Em um lote de **10.000 latas** de refrigerante foi realizada uma amostra aleatória simples de **100 latas** e foi obtido o **desvio padrão amostral do conteúdo das latas igual a 12 ml**. O fabricante estipula um **erro máximo sobre a média populacional de apenas 5 ml**. Para garantir um **nível de confiança de 95%** qual o tamanho de amostra deve ser selecionado para este estudo?

## <font color=green>5.2 Variáveis quantitativas e população finita</font>
***

#### Com desvio padrão conhecido

## $$n = \frac{z^2 \sigma^2 N}{z^2 \sigma^2 + e^2(N-1)}$$

#### Com desvio padrão desconhecido

## $$n = \frac{z^2 s^2 N}{z^2 s^2 + e^2(N-1)}$$

Onde:

$N$ = tamanho da população

$z$ = variável normal padronizada

$\sigma$ = desvio padrão populacional

$s$ = desvio padrão amostral

$e$ = erro inferencial

## <font color='blue'>Exemplo: Indústria de refrigerantes</font>

Em um lote de **10.000 latas** de refrigerante foi realizada uma amostra aleatória simples de **100 latas** e foi obtido o **desvio padrão amostral do conteúdo das latas igual a 12 ml**. O fabricante estipula um **erro máximo sobre a média populacional de apenas 5 ml**. Para garantir um **nível de confiança de 95%** qual o tamanho de amostra deve ser selecionado para este estudo?

### Obtendo $N$

### Obtendo $z$

### Obtendo $s$

### Obtendo $e$

### Obtendo $n$

## $$n = \frac{z^2 s^2 N}{z^2 s^2 + e^2(N-1)}$$

# <font color=green>6 FIXANDO O CONTEÚDO</font>
***

## <font color='blue'>Exemplo: Rendimento médio</font>

Estamos estudando o **rendimento mensal dos chefes de domicílios com renda até R$\$$ 5.000,00 no Brasil**. Nosso supervisor determinou que o **erro máximo em relação a média seja de R$\$$ 10,00**. Sabemos que o **desvio padrão populacional** deste grupo de trabalhadores é de **R$\$$ 1.082,79** e que a **média populacional** é de **R$\$$ 1.426,54**. Para um **nível de confiança de 95%**, qual deve ser o tamanho da amostra de nosso estudo? Qual o intervalo de confiança para a média considerando o tamanho de amostra obtido?

### Construindo o dataset conforme especificado pelo problema

### Calculando o tamanho da amostra

### Calculando o intervalo de confiança para a média

### Realizando uma prova gráfica